src/components/Avatar/readme.md

```diff
@@ -95,16 +95,9 @@ import { Avatar } from 'react-rainbow-components';
 import React from 'react';
 import { Avatar } from 'react-rainbow-components';
 
-const avatarContainerStyles = {
-    borderRadius: '0 0 0.875rem 0.875rem',
-};
-
-<div
-    className="rainbow-p-vertical_x-large rainbow-p-left_medium rainbow-background-color_dark-1"
-    style={avatarContainerStyles}
->
+<InverseContainer className="rainbow-p-vertical_large rainbow-align-content_center rainbow-flex_wrap">
```

Comment:
we should keep all examples align to the left in order to maintain visual consistency